docs/reference/indices/analyze.asciidoc

```diff
@@ -18,6 +18,19 @@ curl -XGET 'localhost:9200/_analyze' -d '
 
 coming[2.0.0, body based parameters were added in 2.0.0]
 
+If text parameter is provided as array of string, it analyze as a multi-valued.
```

Comment:
I would change this:

`If text parameter is provided as array of string, it analyze as a multi-valued.`

to `If text parameter is provided as array of strings, it is analyzed as a multi-valued field.`